packages/console/src/hooks/use-user-default-tenant-id.ts

```diff
@@ -17,32 +17,40 @@ const key = 'defaultTenantId';
 const useUserDefaultTenantId = () => {
   const { data, update: updateMeCustomData } = useMeCustomData();
   const { tenants, currentTenantId } = useContext(TenantsContext);
+  const storedId = useMemo(
+    () => trySafe(() => z.object({ [key]: z.string() }).parse(data)[key]),
+    [data]
+  );
```

Comment:
Use zod `safeParse`?

Reply:
need to check `success`, kind of ugly

Reply:
OK, then perhaps zod `.parse()`? This one doesn't require the success check

Reply:
Will the custom data have other string fields in the future?